stravalib/model.py

```diff
@@ -494,8 +728,13 @@ def __repr__(self):
         )
 
 
+# TODO: this wasn't working as intended - test again
```

Comment:
Once we have addressed #390 (which may be user error) i can test this again. it was not returning correct data because i was querying a activity via a date and getting a totally different subset of items back.